### PR TITLE
Add card paragraph bundle

### PR DESCRIPTION
**Pull request by jordan-caldwell**
## Summary
This PR includes config and theming for the new card paragraph.

Also included is a fix from last deploy where a permission was missing. 

## Metadata

| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes
| Did you apply meaningful labels to the pull request? | yes
| Did you perform a self review first? | yes
| `CHANGELOG` reflects changes? | yes
| Did you perform browser testing? | no
| Risk level | Low
| Relevant links | JIRA issue, bug reports, security alerts, etc.

### CODE REVIEW

File: ecms_base/config/install/user.role.anonymous.yml [hunk 1]
```diff
@@ -16,4 +16,5 @@ permissions:
   - 'view paragraph content icon_card'
   - 'view paragraph content media_item'
   - 'view paragraph content promotion_reference'
+  - 'view paragraph content publication_list'
```
Comment (pfrilling) [inline ecms_base/config/install/user.role.anonymous.yml:19]: I added this with RIG-178. A rebase should remove this.

Comment (pfrilling) [reply ecms_base/config/install/user.role.anonymous.yml:19]: Also, is the 'card' paragraph new? We need to add the appropriate permissions to the anon/auth/author/publisher/admin users

**Review by pfrilling — CHANGES_REQUESTED** [review]
One quick change and a question.

File: ecms_base/config/install/user.role.anonymous.yml (hunk 1 above)
Comment (jordan-caldwell) [reply ecms_base/config/install/user.role.anonymous.yml]: Think I got all of them now

File: ecms_base/config/install/user.role.site_admin.yml [hunk 2]
```diff
@@ -116,6 +117,7 @@ permissions:
   - 'delete own webform'
   - 'delete own webform content'
   - 'delete paragraph content accordion_builder'
+  - 'delete paragraph content card'
```
Comment (pfrilling) [inline ecms_base/config/install/user.role.site_admin.yml:120]: Missing the `update` permission for the site_admin.

**Review by pfrilling — CHANGES_REQUESTED** [review]
One quick change for the site_admin role.

**Review by pfrilling — APPROVED** [review]
🎇. Looks good.

**Pull request merged**